corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -411,10 +411,8 @@ class CaseAccessorSQL:
 
     @staticmethod
     def get_case(case_id):
-        try:
-            return CommCareCase.objects.partitioned_get(case_id)
-        except CommCareCase.DoesNotExist:
-            raise CaseNotFound(case_id)
+        """DEPRECATED"""
```

Comment:
Would it be worth using `warn()`?
```suggestion
        warnings.warn(
            'CaseAccessorSQL.get_case() is deprecated. Please use '
            'CommCareCase.objects.get_case()',
            DeprecationWarning
        )
```
As an added bonus, PyCharm (and probably VS Code too) will display the method with a strike through, and the warning as a tooltip.

Reply:
I don't think so. The plan is to delete these on fairly a short time-scale, as soon as they are not used by any more code. Hopefully that will be within a week or so.

Reply:
https://github.com/dimagi/commcare-hq/pull/31060/commits/9b6b7d300a5b25d5c5e03736c8c1cb132ac8ec40